### DISCUSSION TITLE
Firmware update - Docker?

### DISCUSSION THREAD

**Discussion by MartinBachmannHD** [OpenCCU/OpenCCU#2438] [🇩🇪 Fragen&Antworten] [answered]
Hallo,
ich verwende die aktuelle RaspberryMatic Docker-Version via HMIP-RFUSB auf einem RPi4. Läuft sehr gut! Irgendwie scheine ich aber das Firmeware-Updaten nicht hinzubekommen. Im entsprechenden Menüeintrag steht, dass alles ok ist - alle sind up-to-date. Wenn ich aber hier nachsehe: https://ccu3-update.homematic.com/firmware/api/firmware/search/DEVICE dann sehe ich dass, z.B. mein HmIP-eTRV-B bei mir auf 1.0.20 ist - es würde aber schon eine Version 1.2.12 geben - also anscheinend etwas größeres. 
Jetzt frage ich mich ob es z.B. am Docker-Image liegt (Port oder so geschlossen?) dass der automatische Download nicht klappt... Wie kann ich das denn sehen? 
Danke und schöne Grüße!

**Comment by Baxxy13** [comment, marked as answer]
HmIP-eTRV-B ist nicht gleich HmIP-eTRV-B-2

Und automatische Downloads gibt's nicht und gab es auch nie.

Ich würde sagen... alles ok.

**Reply by MartinBachmannHD** [reply]
Oh.. Da hast Du recht. Ok, danke für die schnelle Antwort
